File: ContactBook/ContactBook.Web/Controllers/HomeController.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using http = System.Web.Http;
using System.Web.Mvc;
using Microsoft.AspNet.Identity;

namespace ContactBook.Web.Controllers
{
    [Authorize]
    public class HomeController : Controller
    {
        public ActionResult Index()
        {
            return View();
        }
    }
}

File: ContactBook/ContactBook.Web/app/controllers/homeController.js
(function () {
    'use strict';

    angular.module('contactbook.controllers').controller("homeController",
    ['$scope', '$location', 'authenticationSvc',
        function ($scope, $location, authenticationSvc) {
            $scope.homeLogout = function () {
                authenticationSvc.logout();
            };
        }
    ]);

})();
